files.js

```diff
@@ -0,0 +1,199 @@
+module.exports = {
+  "/": `<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>The Elements</title>
+    <link rel="stylesheet" href="/css/styles.css">
+  </head>
+  <body>
+    <h1>The Elements</h1>
+    <h2>These are all the known elements.</h2>
+    <h3>These are 2</h3>
+    <ol>
+      <li>
+        <a href="/hydrogen.html">Hydrogen</a>
+      </li>
+      <li>
+        <a href="/helium.html">Helium</a>
+      </li>
+    </ol>
+  </body>
+  </html>`,
+  "/index.html": `<!DOCTYPE html>
```

Comment:
Rather than having an entry for both `/` and `/index.html` could you handle this in code instead?